samples/CloudClipboard/CloudClipboard/GarbageCollector/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Identity;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+
+namespace GarbageCollector
+{
+    public static class Program
+    {
+        private static readonly Uri BlobServiceUri = new Uri("https://cloudclips.blob.core.windows.net/");
+
+        public static async Task Main()
+        {
+            BlobClientOptions options = new BlobClientOptions();
+            options.Retry.MaxRetries = 10;
+            options.Retry.Delay = TimeSpan.FromSeconds(3);
+            options.Diagnostics.IsLoggingEnabled = true;
+
```

Comment:
nit:  Extra blank lines.   If these are intended for emphasis, I'd suggest that comments would be more appropriate to highlight why they're important.